README update for packages (#771)
microsoftgraph/msgraph-sdk-typescript: packages/msgraph-sdk-admin/README.md

> # `@microsoft/msgraph-sdk-admin`

Author: Should be duplicated in all the subfolders

Get started with the Microsoft Graph SDK for TypeScript by integrating the [Microsoft Graph API](https://docs.microsoft.com/graph/overview) into your TypeScript application!

> > **Note:** This package is a complementary package that allows you to build applications using the [v1.0](https://docs.microsoft.com/graph/use-the-api#version) of Microsoft Graph. If you want to try the latest Microsoft Graph APIs, use our [beta SDK](https://github.com/microsoftgraph/msgraph-beta-sdk-typescript) instead.

Reviewer: Again, I recommend using the `[!NOTE]` syntax for your quote blocks. I'd also avoid using potentially confusing words like `complementary` - I think you're saying that it complements the main SDK, is that right? If so, I'd combine this with the note block you have a few lines down and simplify the language. Something like:

```suggestion
> [!NOTE]
> This package requires the `@microsoft/msgraph-sdk` package. It allows you to build applications using the [v1.0](https://learn.microsoft.com/graph/use-the-api#version) of Microsoft Graph. If you want to try the latest Microsoft Graph APIs, use our [beta SDK](https://github.com/microsoftgraph/msgraph-beta-sdk-typescript) instead.
```

# this will install the fluent API package for the administrative API paths
npm install @microsoft/msgraph-sdk-admin
```

## Usage

Here is an example of how to use the package:

```typescript
import { createGraphServiceClient, GraphRequestAdapter } from "@microsoft/msgraph-sdk";
import { AzureIdentityAuthenticationProvider } from "@microsoft/kiota-authentication-azure";
import { ClientSecretCredential } from "@azure/identity";
import "@microsoft/msgraph-sdk-admin"

// Create a credential using ClientSecretCredential
const credential = new ClientSecretCredential(
  'YOUR_TENANT_ID',
  'YOUR_CLIENT_ID',
  'YOUR_CLIENT_SECRET',
);

// Create an authentication provider
const authProvider = new AzureIdentityAuthenticationProvider(credential, ["https://graph.microsoft.com/.default"]);

// Create a request adapter
const requestAdapter = new GraphRequestAdapter(authProvider);

// Create a GraphServiceClient
const graphServiceClient = createGraphServiceClient(requestAdapter);

// Use the GraphServiceClient to make requests by including the appropriate API calls package
// e.g graphServiceClient.admin.(complete your call)

```

## Documentation

For more detailed documentation, see:

> * [Overview](https://docs.microsoft.com/graph/overview)

Reviewer: Replace `docs.microsoft.com` with `learn.microsoft.com` - this can be a global find and replace.
